### PR TITLE
Add Control of Majority of Directors and update validation rules for the share percentage inputs

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/17975
*Issue:*https://github.com/bcgov/entity/issues/17988

*Description of changes:*
- Update the validation rules to match the Validation and Messaging Matrix https://1drv.ms/x/s!AlejmOwhCCvlgcMDvsUyEofpuM1a9A?e=57KzZS

- Add new checkboxes for Control of Majority of Directors

- Update the styles for tooltips (within the Control of Majority of Directors and the footer)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-61-2r5xu2cr.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-61-2r5xu2cr.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-61-2r5xu2cr.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-61-2r5xu2cr.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-61-2r5xu2cr.web.app

**Review by hfekete — APPROVED** [review]

File: btr-web/btr-main-app/components/individual-person/control/ControlOfDirectors.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+})
+
+const { t } = useI18n()
+const types = [
+  { value: 'directControl', label: t('texts.controlOfDirectors.directControl') },
+  { value: 'indirectControl', label: t('texts.controlOfDirectors.indirectControl') },
+  { value: 'significantInfluence', label: t('texts.controlOfDirectors.significantInfluence') }
+]
+
+const controlOfDirectors: Ref<ControlOfDirectorsI> = ref(prop.modelValue)
```
[the remark this answers is not in the document]
Comment (kialj876) [reply btr-web/btr-main-app/components/individual-person/control/ControlOfDirectors.vue:60]: Oh hang on looks like I am wrong about this it does update it

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 2]
```diff
@@ -255,6 +281,14 @@ const typeOfControl: Ref<ControlTypeI> = ref({
   indirectControl: false
 })
 
+// control of directors
+const controlOfDirectors: Ref<ControlOfDirectorsI> = ref({
+  directControl: false,
+  indirectControl: false,
+  significantInfluence: false,
+  noControl: true
```
[the remark this answers is not in the document]
Comment (patrickpeinanw) [reply btr-web/btr-main-app/components/individual-person/AddNew.vue]: Yes. I agree. I have changed it to false

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-61-2r5xu2cr.web.app

**Pull request merged**